guides/common/modules/proc_increasing-the-logging-level-for-puppet-agent.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="increasing-the-logging-level-for-puppet-agent"]
+= Increasing the logging level for Puppet agent
+
+You can find the logs in `/var/log/puppetlabs/puppet/`.
+
+.Procedure
+ifdef::satellite[]
+. Ensure Puppet is enabled in your {Project}.
```

Comment:
Is this true? Is Puppet agent related to foreman_puppet? Or is it related to the use of Puppet in foreman-installer? @ekohl

Reply:
foreman-installer uses puppet-agent as a package, not as a deamon. So it's more related to foreman_puppet.